engine/engine/content/builtins/materials/sprite_paged_atlas.fp

```diff
@@ -1,12 +1,20 @@
-varying mediump vec2 var_texcoord0;
-varying lowp float   var_page_index;
+#version 140
 
-uniform lowp sampler2DArray texture_sampler;
-uniform lowp vec4           tint;
+in mediump vec2 var_texcoord0;
+in mediump float var_page_index;
+
+out vec4 out_fragColor;
+
+uniform mediump sampler2DArray texture_sampler;
+
+uniform fs_uniforms
+{
+    mediump vec4 tint;
+};
 
 void main()
 {
     // Pre-multiply alpha since all runtime textures already are
-    lowp vec4 tint_pm = vec4(tint.xyz * tint.w, tint.w);
-    gl_FragColor      = texture2DArray(texture_sampler, vec3(var_texcoord0.xy, var_page_index)) * tint_pm;
+    mediump vec4 tint_pm = vec4(tint.xyz * tint.w, tint.w);
+    out_fragColor = texture(texture_sampler, vec3(var_texcoord0.xy, var_page_index)) * tint_pm;
 }
```

Comment:
Artem asked it, I want to ask again to make sure:
It's pretty frequent to have a material where `vp` shader from `builtins`, but `fp` one is custom.
After this change, such materials will have `vp` shader version 140 and `fp` stay old. 
Will it work? If we expect it to work, maybe it make sense to have test for it?

Reply:
My other PR has such tests, but both pipelines we will produce valid shaders so I know for a fact this will work :) But in any case since we don't run an actual graphics API for tests I can't really write a unit test for it. Perhaps if we can run metal or something on dedicated hardware? 🤔

Reply:
For me, it's enough if you are sure it works :)
I agree, it's not that easy to test.